enterprise/server/cmd/cache_proxy/cache_proxy.go

```diff
@@ -47,6 +49,8 @@ var (
 	serverType = flag.String("server_type", "cache-proxy", "The server type to match on health checks")
 
 	remoteCache = flag.String("cache_proxy.remote_cache", "grpcs://remote.buildbuddy.dev", "The backing remote cache.")
+
+	headersToPropagate = []string{authutil.APIKeyHeader, authutil.ContextTokenStringKey, "x-buildbuddy-origin", "x-buildbuddy-client", authutil.ClientIdentityHeaderName, bazel_request.RequestMetadataKey}
```

Comment:
nit: could make these 2 strings constants in usageutil.go?

Reply:
yup, done!